templates/expo-template-bare-minimum/ios/Podfile

```
@@ -25,7 +25,7 @@ prepare_react_native_project!
 # }
 # ```
 flipper_config = FlipperConfiguration.disabled
-if ENV['NO_FLIPPER'] == "1" || ENV['CI'] then
+if ENV['NO_FLIPPER'] == "1" || (ENV['CI'] && !ENV['EAS_BUILD']) then
```

```suggestion
if ENV['NO_FLIPPER'] == '1' then
```

hi there! thanks for this fix. i am thinking we could simply remove the `CI` check. generating a debug build with flipper on ci services makes sense to me.

---

I was thinking of that too, as far as i remember that CI check wasnt there on `expo-community-flipper`